infra/resources/modules/commons/api/ts_management/v1/_openapi.yaml

```diff
@@ -439,6 +504,19 @@ components:
       enum:
         - CREATING
         - CREATED
+    TrialCollectionPaginated:
```

Comment:
**suggestion(non-blocking)**: we could create a schema only for paginated response

since in OpenAPI you can use multiple references to define a schema, maybe we could have a schema only for pagination (with `next` and `prev` values) and where there are paginated responses, we can reference this new schema and the schema of the items. 

what do you think?

Reply:
Created PaginatedCollection schema in 20f69ea2658432a88daecb105c2db8f5f1408cac